Formula/elm.rb

```diff
@@ -39,18 +43,18 @@ def install
         "source-directories": [
                   "."
         ],
-        "elm-version": "0.19.0",
+        "elm-version": "0.19.0 <= v < 0.20.0",
```

Comment:
This is different, from what the compiler would create via `elm init`. A version range would normally only be used on `type` of `package`, but this is an `application`. See https://github.com/elm/compiler/blob/master/docs/elm.json/package.md for further reading.